Sources/Remote Logging/Crash Logging/CrashLoggingDataProvider.swift

```diff
@@ -12,6 +12,7 @@ public protocol CrashLoggingDataProvider {
     var currentUser: TracksUser? { get }
     var additionalUserData: [String: Any] { get }
     var shouldEnableAutomaticSessionTracking: Bool { get }
+    var performanceTracking: PerformanceTracking { get }
 }
```

Comment:
I was surprised by `CrashLoggingDataProvider` being a `protocol`. I opened #221 to discuss.